spec/models/family_spec.rb

```diff
@@ -5,4 +5,18 @@
     it { is_expected.to validate_presence_of(:last_name) }
     it { is_expected.to validate_presence_of(:capability) }
   end
+
+  describe "scope" do
```

Comment:
I would change this block to be:
```
describe ".active" do
  it...
end

describe ".inactive" do
  it ...
end
```

Reply:
Done previously (I'm sorry I can't find the sha :) )